Disable auto updates
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 meshlab (1.3.3) precise; urgency=low
 .
   * new snapshot release
Author: Stefan Schlager <zarquon42@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- meshlab-1.3.3.orig/meshlab/src/meshlab/mainwindow_Init.cpp
+++ meshlab-1.3.3/meshlab/src/meshlab/mainwindow_Init.cpp
@@ -1112,7 +1112,7 @@ void MainWindow::saveRecentFileList(cons
 
 	if(loadedMeshCounter-lastComunicatedValue>connectionInterval && !myLocalBuf.isOpen())
 	{
-#if not defined(__DISABLE_AUTO_STATS__)
+#if not defined(__ENABLE_AUTO_STATS__)
 		checkForUpdates(false);
 #endif
 		int congratsMeshCounter = settings.value("congratsMeshCounter",50).toInt();
